flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/factories/TestValuesTableFactory.java

```diff
@@ -313,6 +315,9 @@ private static RowKind parseRowKind(String rowKindShortString) {
     private static final ConfigOption<Boolean> BOUNDED =
             ConfigOptions.key("bounded").booleanType().defaultValue(false);
 
+    private static final ConfigOption<Boolean> FINITE =
+            ConfigOptions.key("finite").booleanType().defaultValue(true);
```

Comment:
I wonder if `terminating` would have been a better word. Because finite sounds like a synonym to bounded. Where terminating refers to the runtime behavior. But just a suggestion.